src/router/auth/index.ts

```diff
@@ -10,40 +10,42 @@ import { AuthService } from "./auth.service";
 export class AuthRouter {
   @HttpErrorException
   @DBMiddleware.connectTypeOrm
-  static async login(
+  static async signUp(
     event: APIGatewayEventIncludeConnectionName,
-    __: any,
-    ___: Function,
+    _: any,

```

Comment:
```suggestion
    _context: any,

``` 형식으로 바꿔주세요 가독성이 떨어집니다

Reply:
그래서 제거했습니다